README.md

```diff
@@ -6,7 +6,7 @@ Create query test cases quickly. For UBC CPSC 310 students. [demo](https://www.y
 
 ## Usage
 
-1. Go to [the code](https://raw.githubusercontent.com/ianmah/310-bookmarklet/master/bookmarklet.js), select all, and drag the code to your bookmarks bar. (Or create a new bookmark with content of the file as the URL).
+1. Drag this link to bookmarks: <a href="javascript:(function()%7Bjavascript%3A(function()%7Bif%20(!document.getElementById(%22generate-button%22))%20%7Bconsole.stdlog%20%3D%20console.log.bind(console)%3Bconsole.logs%20%3D%20%5B%5D%3Bconsole.log%20%3D%20function()%7Bconsole.logs.push(Array.from(arguments))%3Bconsole.stdlog.apply(console%2C%20arguments)%3B%7D%3Bconst%20pollForElement%20%3D%20(elem%2C%20timeout%2C%20callback)%20%3D%3E%20%7Bconst%20intervalPoll%20%3D%20setInterval(()%20%3D%3E%20%7Bconst%20element%20%3D%20document.querySelector(elem)%3Bif%20(element)%20%7BclearInterval(intervalPoll)%3Bcallback()%3B%7D%3B%7D%2C%2069)%3BsetTimeout(()%20%3D%3E%20clearInterval(intervalPoll)%2C%20timeout)%3B%7D%3B%2F*%20https%3A%2F%2Fstackoverflow.com%2Fquestions%2F33855641%2Fcopy-output-of-a-javascript-variable-to-the-clipboard%20*%2Fconst%20copyToClipboard%20%3D%20(text)%20%3D%3E%20%7Bvar%20dummy%20%3D%20document.createElement(%22textarea%22)%3Bdocument.body.appendChild(dummy)%3Bdummy.value%20%3D%20text%3Bdummy.select()%3Bdocument.execCommand(%22copy%22)%3Bdocument.body.removeChild(dummy)%3B%7D%3Bconst%20onClick%20%3D%20()%20%3D%3E%20%7Bdocument.getElementById(%22submit-button%22).click()%3BpollForElement(%22.summary%22%2C%203000%2C%20()%20%3D%3E%20%7BsetTimeout(()%20%3D%3E%20%7Bconst%20queryLog%20%3D%20console.logs%5B1%5D%3Bconst%20resultLog%20%3D%20console.logs%5B2%5D%3Bconsole.logs%20%3D%20%5B%5D%3Bconst%20query%20%3D%20queryLog%5B0%5D%3Bconst%20result%20%3D%20resultLog%20%26%26%20resultLog%5B1%5D%3Bconst%20obj%20%3D%20result%20%3F%7Btitle%3A%20%22generated%20by%20https%3A%2F%2Fgithub.com%2Fianmah%2F310-bookmarklet.%20Pls%20gib%20star%22%2Cquery%2CisQueryValid%3A%20true%2C...result%7D%20%3A%20%7Btitle%3A%20%22generated%20by%20https%3A%2F%2Fgithub.com%2Fianmah%2F310-bookmarklet.%20Pls%20gib%20star%22%2Cquery%2CisQueryValid%3A%20false%2Cresult%3A%20%22InsightError%22%7D%3Bconst%20json%20%3D%20JSON.stringify(obj%2C%20null%2C%204)%3BcopyToClipboard(json)%3Bconsole.logs%20%3D%20%5B%5D%3B%7D%2C%20420)%3B%7D)%3B%7D%3Bconst%20parent%20%3D%20document.getElementById(%22submit-container%22)%3Bconst%20button%20%3D%20document.createElement(%22a%22)%3Bbutton.innerHTML%20%3D%20%22Copy%20Test%20File%22%3Bbutton.id%20%3D%20%22generate-button%22%3Bbutton.className%20%3D%20%22btn%22%3Bbutton.href%20%3D%20%22%23%22%3Bbutton.onclick%20%3D%20onClick%3Bparent.appendChild(button)%3B%7D%7D)()%7D)()">310-bookmarklet</a>
```

Comment:
This link isn't clickable for me, when I look at it here: https://github.com/joshflchan/310-bookmarklet/tree/improve-bookmark-link-and-fix-typo

Reply:
Looks like it is only working for me locally 
![image](https://user-images.githubusercontent.com/34149667/93161624-c5762300-f6c7-11ea-888a-2ea8d57c1fb9.png)

Reply:
Have you tried the `[text](link)` syntax?

Reply:
It's doomed https://stackoverflow.com/questions/14951321/how-to-display-html-content-in-github-readme-md